README.md

```diff
@@ -350,6 +350,14 @@ _Note that prior to release 1.2 of VexCL the `VEX_FUNCTION` macro had different
 interface. That version is considered deprecated but is still available as
 `VEX_FUNCTION_V1`._
 
+Type casting a vector can be done by creating a user defined function. This has the advantage of beeing backend independent. 
```

Comment:
What do you think of replacing the first sentence with `Another example of using a custom function is type-casting a vector`?